release/artifacts.yaml

```diff
@@ -179,3 +179,7 @@
   type: jar-scala
 - target: //triggers/runner:trigger-runner-lib
   type: jar-scala
+- target: //triggers/service:trigger-service
```

Comment:
I wouldn’t publish them to maven. These are executables not supported libraries.